roles/marmotta/tasks/tomcat-marmotta.yml

```diff
@@ -22,34 +23,42 @@
 
 - name: install marmotta
   become: yes
-  command: cp {{ install_path }}/apache-marmotta-{{ marmotta_ver }}/marmotta.war /var/lib/tomcat/webapps/marmotta.war \
-           creates=/var/lib/tomcat/webapps/marmotta.war
+  command: cp {{ install_path }}/apache-marmotta-{{ marmotta_ver }}/marmotta.war {{ tomcat }}/webapps/marmotta.war \
+           creates={{ tomcat }}/webapps/marmotta.war
 
 - name: set ownership on marmotta.war
   become: yes
-  command: chown tomcat:tomcat /var/lib/tomcat/webapps/marmotta.war
+  file: path={{ tomcat }}/webapps/marmotta.war owner=tomcat group=tomcat
 
 - name: set permissions for marmotta_home
   become: yes
   file: path={{ marmotta_home }} owner=marmotta group=marmotta mode=0755 state=directory recurse=yes
 
-- name: set marmotta system config properties (linux)
+- name: set marmotta system config properties
   become: yes
   template: src=system-config.properties.j2 dest={{ marmotta_home }}/system-config.properties \
             owner=tomcat group=tomcat backup=no mode=0600
 
+- name: restart tomcat
+  service: name=tomcat enabled=yes state=restarted
+
+- wait_for: path={{ tomcat }}/webapps/marmotta/WEB-INF/lib/log4j-over-slf4j-1.7.7.jar
```

Comment:
Yeah, Tomcat takes some time to start all three webapps (up to 15 seconds in my experience).  Originally I had `wait_for: port=8080`, but Tomcat starts listening on that port before all webapps are running.  Prior to this change, more often than not Ansible was failing to remove the `.jar`.